docs/explanations/motd_messages.md

```diff
@@ -104,8 +135,59 @@ coverage for your applications.
 Your grace period will expire in 9 days.
 ```
 
-### How are these messages updated and inserted into MOTD?
+## MOTD for the ESM announcement
+
+There was also a general announcement info that further helped to raise
+awareness of Pro being available and for free. This was only shown on systems
+covered by `esm-apps`. Furthermore it was meant to be only shown for a limited
+time and therefore removed in 27.14.
+It looked like:
+
+```
+ * Introducing Expanded Security Maintenance for Applications.
+   Receive updates to over 25,000 software packages with your
+   Ubuntu Pro subscription. Free for personal use.
+
+     https://ubuntu.com/pro
+```
+
+## How are these messages inserted into MOTD and how could I disable them?
+
+Just like there are different aspects to the messages outlined above they
+come from different sources into the MOTD that one sees at login time.
+
+Those messages are considered important to ensure user awareness about
+the free additional security coverage provided by Ubuntu Pro and about
+yet unapplied potential updates in general. Therefore it is not recommended
+to disable them, but if one wants to do that anyway they can be disabled
+selectively via the config files that add them as outlined below.
+
+Removing those is considered a conffile change to customize a program
+and they will stay removed even on future upgrades or re-installations of the
+related packages. If you realize that you actually need them back you need
+to reinstall the related packages and tell apt/dpkg to offer you to restore
+those files via:
+`sudo apt install --reinstall -o Dpkg::Options::="--force-confask" ubuntu-advantage-tools update-notifier-common`
```

Comment:
```suggestion
related packages. If you later realise that you want them back, you will need
to reinstall the related packages and tell `apt`/`dpkg` to offer you to restore
those files using the following command:

```
sudo apt install --reinstall -o Dpkg::Options::="--force-confask" ubuntu-advantage-tools update-notifier-common```
```

Reply:
Well...that didn't work as successfully as I'd hoped :P

Reply:
Not sure why it wouldn't let me add a code block there, but for the sake of tidiness I like to have code snippets on a separate line (especially if they're long snippets)